change/@ni-nimble-components-be528419-2f60-4d4b-b6f0-a8b8e1e79bae.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Once we have agreement on the pattern for adding a react wrapper, we should update README and CONTRIBUTING and issue templates to remind us to create it (and explain how to create it) when building a new component.